packages/serve/src/startDevServer.ts

```diff
@@ -11,21 +11,25 @@ import Server from 'webpack-dev-server/lib/Server';
  */
 export default function startDevServer(compiler, options): void {
     const firstWpOpt = compiler.compilers ? compiler.compilers[0].options : compiler.options;
-    const devServerOptions = firstWpOpt.devServer || {};
+    let devServerOptions = firstWpOpt.devServer || {};
```

Comment:
This is not entirely true. Second compiler also can contains `devServer`. To be honestly both can contain `devServer`, we need to think how to handle it, do we have `--name` option to choose compiler?

Reply:
What would be the merge strategy?

Reply:
Assuming you are referring to merger of CLI options with config then I think you may use same strategy for each complier. Just CLI provided options would be same for each of them.

Reply:
I think we should look at all compilers, if you have two `devServer` output `warning` about you need specify `name` of compiler, in future we will dev server as plugin and we don't need this hack